openupgrade_scripts/scripts/stock/16.0.1.1/upgrade_analysis.txt

```diff
@@ -1,11 +1,11 @@
 ---Models in module 'stock'---
-obsolete model stock.location.route
-obsolete model stock.production.lot
+obsolete model stock.location.route (renamed to stock.route)
+obsolete model stock.production.lot (renamed to stock.lot)
 new model lot.label.layout [transient]
 new model picking.label.type [transient]
-new model stock.lot
+new model stock.lot (renamed from stock.production.lot)
```

Comment:
It seem like the tool are detecting just the model is renamed and do not need to have extra `del` and `new` line above so i assumed this one is correct right?

Reply:
Yes, it's correct, and it's done due to https://github.com/OCA/OpenUpgrade/blob/16.0/openupgrade_scripts/apriori.py#L74

Reply:
Oh i understood, so before this PR we haven't had that so it has extra line below